src/main/java/com/airsquared/blobsaver/Background.java

```diff
@@ -173,10 +195,10 @@ private static void saveBackgroundBlobs(int preset) {
         Preferences presetPrefs = Preferences.userRoot().node("airsquared/blobsaver/preset" + preset);
 //        presetPrefs.put("Saved Versions", "[]");                                                        // for testing
         String identifier;
-        if ("none".equals(presetPrefs.get("Device Model", ""))) {
+        if ("none".equals(presetPrefs.get("Device model", ""))) {
```

Comment:
It's too late to change this, because this will force the user to re-save their presets.

Reply:
OK, I'll revert it

Reply:
Wait, then why are the references to "Device model" have model lower-cased?

Reply:
Idk, but it doesn't matter.

Reply:
What do you mean?

Reply:
![image](https://user-images.githubusercontent.com/12981958/53386668-f94bf500-3937-11e9-9b6f-58928a924256.png)
^^ I can only find usages in which model is lower-cased